.changes/v2.13.0/339-improvements.md

```diff
@@ -0,0 +1,3 @@
+* External network type ExternalNetworkV2 uses elevated API (maximum available from 33.0, 35.0 and 36.0) [GH-339]
```

Comment:
This sentence is geeky :) Could we somehow make it more friendly for the casual reader? Few suggestions:
- "elevated API ..." - maybe add "version"?
- What about explaining what "elevated" means in this context in more detail?
Thanks!

Reply:
I have tried to improve the message. How does it look now?

One thing I am not sure about is if we should put any internal links to docs. I could - but we never done this before I think.

We introduced this generic functionality for API version elevation in previous release so that NSX-T Nat Rules can consume latest API.

There is a function https://github.com/Didainius/go-vcloud-director/blob/faf5992f7e4f4338cc76afff924cdc650af2409c/govcd/openapi_endpoints.go#L107 which is used in our resources and some elevations with explanations:
https://github.com/Didainius/go-vcloud-director/blob/faf5992f7e4f4338cc76afff924cdc650af2409c/govcd/openapi_endpoints.go#L59-L71